_authors/bigsamu.markdown

```diff
@@ -0,0 +1,8 @@
+---
+short_name: bigsamu
+name: Samuel Valdes Gutierrez
+github: bigsamu
+photo: "/assets/media/authors/bigsamu.jpg"
```

Comment:
Please remove the added image and this line.

In the last line of the file with an introduction, please start the line with your name bolded like `**<Name>**`

Reply:
Hi @cwperks. I just made the changes! Thanks for the message!